### ISSUE TITLE
Synchronize content of `README.md` to `index.md` page

### ISSUE THREAD

**Issue by LecrisUT** [spglib/spglib#305]
I believe the contents of the README.md is very helpful to have it in the top page of the web page.
`scikit-build-core` does this by having tags `<!-- SPHINX-START -->` and `<!-- SPHINX-END -->`. We can adopt the same approach and insert it here:
https://github.com/spglib/spglib/blob/4cc2dbb15be7d081bd9d45bbb7772a2400bfcedc/doc/index.md?plain=1#L38-L40

Any thoughts?

**Comment by atztogo** [comment]
> I believe the contents of the README.md is very helpful to have it in the top page of the web page.

I agree. The current top page is like for the people who don't know spglib yet. But now probably most of people who come to the page know spglib already. A more technical front page is possible.

I don't see what you mean the next sentence due to lack of my technical knowledge.

**Comment by LecrisUT** [comment]
> The current top page is like for the people who don't know spglib yet. But now probably most of people who come to the page know spglib already. A more technical front page is possible.

I was thinking about the opposite. Making the front page and the index page both be non-technical, moving the technical details to a separate page. Although the [current frontpage](https://spglib.readthedocs.io/en/latest/) does not have any technical details. Stuff like `Features` and `How-to-cite` are great to have on both pages. Details about what's magnetic symmetry vs cell symmetry, what is the `types` number in the atoms, these should be documented, but on a separate page.

> I don't see what you mean the next sentence due to lack of my technical knowledge.

It's ok, just getting the opinion if this is :+1:

**Comment by lan496** [comment]
The idea to use `start-after` and `end-before` to include documents from `README.md` looks nice to me.

Then, we may add the below in `doc/index.md`:
```markdown
	```{include} ../README.md
	:start-after: <!-- SPHINX-START -->
	:end-before: <!-- SPHINX-END -->
	```
```
and add the below in `README.md`
```markdown

<!-- SPHINX-START -->
Some texts to be included in index.md
<!-- SPHINX-END -->
```

**Comment by LecrisUT** [comment]
Ok then I'll implement that in #257 along with a `CHANGELOG.md` with a similar logic
